pkg/aws/eni/node.go

```diff
@@ -39,6 +39,9 @@ const (
 
 	getMaximumAllocatableIPv4FailureWarningStr = "maximum allocatable ipv4 addresses will be 0 (unlimited)" +
 		" this could lead to ip allocation overflows if the max-allocate flag is not set"
+
+	// insufficientPrefixesInSubnetStr AWS error code for insufficient /28 prefixes in a subnet
+	insufficientPrefixesInSubnetStr = "InsufficientCidrBlocks"
```

Comment:
I'm assuming that the AWS SDK does not define this as a const and so we must define it ourselves?

Reply:
Yes, I don't see these codes defined in the SDK. Only `amazon-vpc-cni-k8s` has it defined [here](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/pkg/ipamd/ipamd.go#L177)